frontend/src/store/app/actions.ts

```diff
@@ -7,3 +13,11 @@ export function initialize(): AppAction {
 export function setLocale(locale: string): AppAction {
   return { type: SET_LOCALE, locale };
 }
+
+export function lostBackendConnection(backendConnection: boolean, clock: Date): AppAction {
+  return { type: BACKEND_CONNECTION_DOWN, clock, backendConnection };
```

Comment:
Do we actually Need the "backendConnection" boolean field in these actions? What do the following actions mean?
```
{
  type: BACKEND_CONNECTION_DOWN,
  backendConnection: true
}
```
```
{
  type: BACKEND_CONNECTION_DOWN,
  backendConnection: false
}
```

(this also applies to the definitions of the `BackendConnectionDown` and `BackendConnectionUp` interfaces in `types.ts`)